### PR TITLE
update community instructions

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Update community build instructions to use SOURCE_URL instead of DOWNLOAD_SRPMS
Remove unneeded values from PACKAGE_IGNORE_LIST

###### Change Log  

- Update documentation

###### Does this affect the toolchain?  

YES
**NO**

###### Test Methodology

- Local builds

### CODE REVIEW

File: toolkit/docs/building/building.md [hunk 1]
```diff
@@ -306,7 +306,7 @@ sudo make image CA_CERT=/path/to/rootca.crt TLS_CERT=/path/to/user.crt TLS_KEY=/
 
 ## Building Everything From Scratch
 
-**NOTE: Source files must be made available for all packages. They can be placed manually in the corresponding SPEC/\* folders, `SOURCE_URL=<YOUR_SOURCE_SERVER>` may be provided, or DOWNLOAD_SRPMS=y may be used to use pre-packages sources**
+**NOTE: Source files must be made available for all packages. They can be placed manually in the corresponding SPEC/\* folders, `SOURCE_URL=<YOUR_SOURCE_SERVER>` may be provided, or DOWNLOAD_SRPMS=y may be used to use pre-packages sources. Core Mariner source packages are available at `SOURCE_URL=https://cblmarinerstorage.blob.core.windows.net/sources/core`**
```
Comment (PawelWMS) [inline toolkit/docs/building/building.md:309]: We could also not here, that it is currently impossible to mention multiple source URLs within one build.

**Review by PawelWMS — APPROVED** [review]

File: toolkit/docs/building/building.md (hunk 1 above)
Comment (anphel31) [reply toolkit/docs/building/building.md:309]: Good point, although I think there is a better place for that note. Updated.

**Review by christopherco — APPROVED** [review]

**Comment by christopherco** [comment]
Thoughts on setting the default `SOURCE_URL ?=` in the Makefile to the blob store? Not needed for this PR. Just curious if this is something we want to do. It'd be nice to not need to add the `SOURCE_URL=` to the make line :-)

**Comment by PawelWMS** [comment]
> Thoughts on setting the default `SOURCE_URL ?=` in the Makefile to the blob store? Not needed for this PR. Just curious if this is something we want to do. It'd be nice to not need to add the `SOURCE_URL=` to the make line :-)

We discussed this briefly and decided against it until we provide support for multiple source URL, or think through the derivative scenario in more detail. If anything, they'd probably be more interested in using their own source servers and not core CBL-Mariner's storage, so it might be a better idea to use/hard-code the core blob storage separately from `SOURCE_URL`.

**Comment by christopherco** [comment]
> > Thoughts on setting the default `SOURCE_URL ?=` in the Makefile to the blob store? Not needed for this PR. Just curious if this is something we want to do. It'd be nice to not need to add the `SOURCE_URL=` to the make line :-)
> 
> We discussed this briefly and decided against it until we provide support for multiple source URL, or think through the derivative scenario in more detail. If anything, they'd probably be more interested in using their own source servers and not core CBL-Mariner's storage, so it might be a better idea to use/hard-code the core blob storage separately from `SOURCE_URL`.

Makes sense. Thanks for explaining!

**Pull request merged**